src/common/__tests__/BaseAnnotator-test.ts

```diff
@@ -130,10 +130,10 @@ describe('BaseAnnotator', () => {
     });
 
     describe('setVisibility()', () => {
-        test.each([true, false])('should dispatch setVisibilityAction with visibility %p', visibility => {
+        test.each([true, false])('should hide/show annotations if visibility is %p', visibility => {
+            annotator.rootEl = defaults.container;
```

Comment:
Probably just call `annotator.init(1)` to set up the annotator properly.

Reply:
> Probably just call `annotator.init(1)` to set up the annotator properly.

If call `init` but not set `rootEl` explicitly, eslint will throw error `annotator.rootEl is possibly null`